hue/hue/base/hive-site-xml-secret.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hue-hive-site-xml
+type: Opaque
+stringData:
+  hive-site.xml: |
+    <?xml version="1.0"?>
+    <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+    <configuration>
+      <property>
+        <name>hive.server2.transport.mode</name>
+        <value>binary</value>
+        <description>Server transport mode. binary or http.</description>
+      </property>
+
+      <property>
+        <name>hive.server2.thrift.http.port</name>
+        <value>10000</value>
+        <description>Port number when in HTTP mode.</description>
+      </property>
+
+      <property>
+        <name>fs.s3a.aws.credentials.provider</name>
+        <value>com.amazonaws.auth.EnvironmentVariableCredentialsProvider</value>
+        <description>
+          Comma-separated class names of credential provider classes which implement
+          com.amazonaws.auth.AWSCredentialsProvider.
+
+          These are loaded and queried in sequence for a valid set of credentials.
+          Each listed class must implement one of the following means of
+          construction, which are attempted in order:
+          1. a public constructor accepting java.net.URI and
+              org.apache.hadoop.conf.Configuration,
+          2. a public static method named getInstance that accepts no
+            arguments and returns an instance of
+            com.amazonaws.auth.AWSCredentialsProvider, or
+          3. a public default constructor.
+
+          Specifying org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider allows
+          anonymous access to a publicly accessible S3 bucket without any credentials.
+          Please note that allowing anonymous access to an S3 bucket compromises
+          security and therefore is unsuitable for most use cases. It can be useful
+          for accessing public data sets without requiring AWS credentials.
+
+          If unspecified, then the default list of credential provider classes,
+          queried in sequence, is:
+          1. org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider:
+            Uses the values of fs.s3a.access.key and fs.s3a.secret.key.
+          2. com.amazonaws.auth.EnvironmentVariableCredentialsProvider: supports
+              configuration of AWS access key ID and secret access key in
+              environment variables named AWS_ACCESS_KEY_ID and
+              AWS_SECRET_ACCESS_KEY, as documented in the AWS SDK.
+          3. com.amazonaws.auth.InstanceProfileCredentialsProvider: supports use
+              of instance profile credentials if running in an EC2 VM.
+        </description>
+      </property>
+
+      <property>
+        <name>fs.s3a.awsAccessKeyId</name>
+        <value>AWS access key ID</value>
+        <description>Enables S3 browser. Safety valve.</description>
+      </property>
+
+      <property>
+        <name>fs.s3a.awsSecretAccessKey</name>
+        <value>AWS secret key</value>
```

Comment:
It's missing the parameter to specify the AWS Secret key from params.env

Reply:
This PR is WIP for a reason. Please don't review now.

Reply:
Btw. The aim is to load them from environment, see above. It can't be specified here as inlined values, since that would require us traversing vars multiple times and kustomize can't do that. (a parameter defines which secret holds the s3 credentials -> loaded into vars -> based on this var we load another secret and store the creds into another variables and then template this file, that's simply not possible).

It should be doable to load it from the environment, so don't mind these values at all. I should be able to bypass it.